tap_square/client.py

```diff
@@ -207,6 +207,33 @@ def get_inventories(self, variation_ids, start_time):
 
             yield (result.body.get('counts', []), result.body.get('cursor'))
 
+    def get_shifts(self, start_time):
+        body = {
+            "query": {
+                "sort": {
+                    "field": "UPDATED_AT",
+                    "order": "ASC"
+                }
+            }
+        }
+        with singer.http_request_timer('GET shifts'):
+            result = self._client.labor.search_shifts(body=body)
+
+        if result.is_error():
+            raise Exception(result.errors)
+
+        yield (result.body.get('shifts', []), result.body.get('cursor'))
```

Comment:
I don't like the pattern of doing the yield twice. It seems unnecessary. But we need to refactor a ton of this so it's not a high priority right this second.

Reply:
I agree that we can change this later